src/index.html

```diff
@@ -1,12 +1,486 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+      href="/styles/sass/styles.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
+
   <body>
-    <h1>Catalog</h1>
+
+    <!-- #region HEADER -->
+    <header class="header">
+      <!-- <div class="header__logo-container"> -->
+      <a href="#home" class="header__logo-link">
+        <img class="header__logo-picture" src="./images/logo.png"
+          alt="moyo-logo">
+      </a>
+      <!-- </div> -->
+
+      <nav class="nav">
+        <div class="nav__list">
+          <li>
+            <a href="#Apple" class="nav__list-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a href="#Samsung" class="nav__list-link">Samsung </a>
+          </li>
+
+          <li>
+            <a href="#Smartphones" class="nav__list-link">Smartphones </a>
+          </li>
+
+          <li>
+            <a data-qa="nav-hover"
+              class="nav__list-link"
+              href="#Laptops & Computers"
+              >
```

Comment:
```suggestion
            <a 
              data-qa="nav-hover"
              class="nav__list-link"
              href="#Laptops & Computers"
            >
```
[code style rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Fix everywhere

![image](https://github.com/mate-academy/layout_catalog/assets/86923089/1ab3d36f-f3ea-43d7-8146-e2c943ea842e)